src/graphql/db/index.ts

```diff
@@ -0,0 +1,42 @@
+import { User } from "../../schema"
```

Comment:
I'm not sure about the place of this file, if it's just a first step for the repository refactor that's fine, but I'm not aware if anything else has been defined.
cc @bodymindarts

Reply:
Yeah, we're gonna need a big refactor here to unify all db calls in one place.

Reply:
AFAICT this file defines top level functions that get called from mutations.
I like the pattern of how the user is accessed and updated.
I don't like that this file is in the scoped under `graphql/db`. If GraphQL is a layer for interacting with an application then these functions represent the application entrypoints.

This file should be something like:
```
src/app/user.ts
```
or
```
src/usecases/user.ts
```
Actually for the GQL redesign I think it would be great to introduce this kind of structure in general. Functions that get called as app-entrypoints scoped in an expressive folder structure. Should be used both for queries and mutations. Calling the database directly from a graphql resolver should be avoided.